### DISCUSSION TITLE
Status LED implementation

### DISCUSSION THREAD

**Discussion by pavelyazev** [open-sdr/openwifi#436] [General]
Hi,

I am working on the porting openwifi to a new platform.
I use Zed board as a reference platform. In the original implementation Zed board blinks by its LEDs during operation.
I would like to reproduce similar behavior.
Unfortunately, I have not managed to find how status led functionality is implemented.
From the device tree and HW project I understand led connections. 
But I have not found which part controls leds from the software side.

Can someone clarify this topic?

**Comment by tryss** [comment]
Hi, the leds comes directly from Tx and Rx activities.

**Reply by pavelyazev** [reply]
Yes, you are right. 
I thought that leds have a software control.
Thank you.
